pubspec.yaml

```diff
@@ -24,10 +24,11 @@ dependencies:
   logging: ^1.0.0
   cli_util: ^0.4.0
   glob: ^2.0.0
-  file: ^6.0.0
+  file: ^7.0.0
   package_config: ^2.1.0
   yaml_edit: ^2.0.3
 
 dev_dependencies:
   lints: ^2.0.1
   test: ^1.16.2
+  json_schema: ^5.1.1
```

Comment:
`json_schema` has a LICENSE that I personally am not familiar with: https://pub.dev/packages/json_schema/license

I don't know what are the implications of using it but I think we should investigate. cc/ @dcharkes